### ISSUE TITLE
Address Change Workflow 

### ISSUE THREAD

**Issue by rachaelwoolen** [seedco/Workshop#19]
### What
Complete internal address change workflow

### Description
There are still a few holes in our address change workflow. This ticket is an attempt to walk through the entire scenario so we can solidify this process as soon as possible. This is what I have documented in the [Knowledge Base](https://docs.google.com/document/d/15OoSMe0XFCqfK2KEd70owTdnyU5tM0ygMcfsPwzcYcE/edit) so far, but I still have questions about several steps:

1. The customer must request the address change in writing via support message (with a signature - sent to Bancorp by compliance)
2. Address changes must be made manually through Engineering.
3. Address must be verified in IDM before the update can be made
4. If address cannot be verified, customer must submit documents (see Acceptable Verifying Docs)
5. Once the customer has submitted a request, make a ticket in Github with the following information in the Highpri repo, tag compliance and the on-call engineer, then ping the on-call engineer in #oncall:
- Last four digits of account number 
- Name 
- Old address and new address - Specify if we are changing the personal or business address. 

### User Story
As a support agent, here is the process I would go through to update an address: 

*1. The customer must request the address change in writing via support message (with a signature, to be sent to Bancorp by compliance) 
- I don't fully understand this requirement and I feel that I need to in order to make sure it is as efficient as possible and to answer any questions that a customer may have about the process. 
- Why does the bank need the request in writing? 
- Is a Zendesk request enough?
- If we already have the signature on file, why would we need to resend it to TBBK?
- Is there an address change form that we need to have the customer fill out or that we are filling out for them?
- If we can update the address via api, what is the purpose of the form or written request?

*2. Address changes must be made manually through Engineering.* 
- I understand that TBBK has an API that we can use, can it be hooked up to Teg so that address changes are set to the bank automatically? 

*3. Address must be verified in IDM before the update can be made*
- Can Support get access to IDM ASAP?

*4. If the address cannot be verified, customer must submit documents (see Acceptable Verifying Docs)*
- Do we need to verify the address just for an update, even if the customer is requesting no other action? 
- Since cards can only be sent to a residential address, why would ever need to verify the business address?

*5. Once the customer has submitted a request, make a ticket in Github with the following information in the Highpri repo, tag compliance and the on-call engineer, then ping the on-call engineer in #oncall:*
- Last four digits of account number 
- Name 
- Old address and new address - Specify if we are changing the personal or business address. 

- @benjisg Assuming we continue with manual address updates, is this all the information that you need? 

cc @lh12345 @melgeyer @benjisg

**Comment by benjisg** [comment]
@rachaelwoolen yep that should be it! The main distinction we need is Business Address vs Card Address. We've been saying "business vs personal", but right now we equate:

* `Individual Address --> Card Address`
* `Member Address --> Business Address`

So we don't really have a "personal" address per se; I believe the primary side effect of this will be the address that account statements are sent to by Bancorp. I haven't heard whether they use the address associated with the card or the customer record for these

**Comment by rachaelwoolen** [comment]
@benjisg Thanks for the clarification, I'll get that updated in our docs.

**Comment by lh12345** [comment]
@rachaelwoolen Just so I'm following - the bus. address will still be tied to the member (bus.) and the personal address will still be tied to the individual (owner/auth. signer), correct?  Change request for bus. address will also be verified similar to the address change for personal address.
